tests/e2e/e2e.go

```diff
@@ -252,7 +252,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	pvList, err := kubeCli.CoreV1().PersistentVolumes().List(metav1.ListOptions{})
 	framework.ExpectNoError(err, "failed to list persistent volumes")
 	for _, pv := range pvList.Items {
-		if pv.Spec.StorageClassName != "local-storage" {
+		if pv.Spec.StorageClassName != "standard" {
```

Comment:
What if we add local PV case later? Is it necessary to check the SC here? Or we can just skip this and check the reclaim policy?

Reply:
fixed